id-repository/id-repository-identity-service/configure_start.sh

```
@@ -4,7 +4,15 @@
 set -e
 
 echo "Downloading pre-requisites install scripts"
+
 wget --no-check-certificate --no-cache --no-cookies $artifactory_url_env/artifactory/libs-release-local/deployment/docker/id-authentication/configure_biosdk.sh -O configure_biosdk.sh
+wget -q --show-progress "${kernel_ref_idobjectvalidator_url}" -O "${loader_path_env}"/kernel-ref-idobjectvalidator.jar; \
```

Shellcheck (0.11.0)

[warning] kernel_ref_idobjectvalidator_url is referenced but not assigned.

(SC2154)